.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,40 @@
+# template from https://github.com/JamesIves/github-pages-deploy-action/tree/dev
+
+name: CI
+
+run-name: Deployment Triggered - ${{ github.event.workflow_run.conclusion == 'success'}}
+
+on:
+  workflow_run:
+    workflows: ["Run tests"]
+    types:
+      - completed
+
+jobs:
+  build-and-deploy:
+    #concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
I love GH's `if:` syntax. Sometimes it's hard to figure out what to test for, but it's nicely powerful when we need it.